Sources/Networking/HTTPClient/HTTPRequest.swift

```diff
@@ -147,6 +143,24 @@ extension HTTPRequest.Path {
         }
     }
 
+    // Whether the endpoint will perform signature validation.
+    var hasSignatureValidation: Bool {
```

Comment:
Maybe rename to `supportsSignatureValidation`?

Reply:
That's better 👍🏻